db/pg/index.js

```diff
@@ -2,14 +2,15 @@
 
 const path = require('path');
 
-const log = require('../../server/lib/logger');
+const { Logger } = require('../../server/lib/logger');
 
 const massive = require('massive');
 const pgConn = require('pg-connection-string');
 
 const { DB } = require('config');
 
 const MODULE_ID = path.relative(process.cwd(), module.id) || require(path.resolve('./package.json')).name;
```

Comment:
```
const MODULE_ID = path.relative(process.cwd(), module.id) || require(path.resolve('./package.json')).name;
```
is repeated in all the files that call the logger. This can be moved into the logger class. The logger class would be instantiated like this, passing in the module.id.

```
const log = new Logger({sourceID: module.id});
```

Reply:
Unfortunately not, there are many files without the `MODULE_ID`.

For example all the files in `/controllers/*`

https://github.com/Financial-Times/next-syndication-api/pull/264/files#diff-e25960b81c24078fa56a96ecbdf30b70a9d69ea20b5bc99e8ba3d8c15ab8d507R12